src/css/styles.css

```diff
@@ -625,6 +625,15 @@ html {
 
 /********************************* PARADIGM *********************************/
 
+
+.inflection-observed-frequency{
+  /* todo: stub */
+}
+
+.unobserved-inflection-cell{
+  color: gray;
+}
+
```

Comment:
need people with asthetics and accessibility knowledge

Reply:
Haha, for sure! We can tackle this in a future PR.